app/assets/stylesheets/components/common/_global_nav.scss

```diff
@@ -135,6 +135,18 @@ $global-nav-transition-duration: 300ms;
   }
 }
 
+.touch {
+  .global-nav__clump {
+    &:hover {
+      background: transparent;
+    }
+
+    &.is-active {
+      background: $color-white;
+    }
+  }
+}
```

Comment:
Might be worth nesting this slightly different, place .touch within .global-nav__clump:

```
.global-nav__clump {
  .touch & {
    &:hover {
       background: transparent;
     }
 
     &.is-active {
       background: $color-white;
     }
  }
}
```

Just a suggestion to make the css easy to read.